apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/ChatModel.kt

```diff
@@ -1642,7 +1642,7 @@ data class GroupMember (
   fun canChangeRoleTo(groupInfo: GroupInfo): List<GroupMemberRole>? =
     if (!canBeRemoved(groupInfo)) null
     else groupInfo.membership.memberRole.let { userRole ->
-      GroupMemberRole.values().filter { it <= userRole && it != GroupMemberRole.Author }
+      GroupMemberRole.supportedRoles.filter { it <= userRole }
```

Comment:
what does it mean "supported role"? Name suggests that any other role except from supportedRoles is not supported. Please, use naming that describes what it actually means